lenses-freeform.html

```diff
@@ -497,12 +501,13 @@
       this._selWrappers = [];
       this._selElement = null;
     },
-    dragStartResize: function (e, details, selection) {
+    dragStartResize: function (e) {
       e.stopPropagation();
     },
-    dragResize: function (e, details, selection) {
+    dragResize: function (e) {
       var self = this;
-      var wrapper = selection.parentNode;  //var styles = window.getComputedStyle(wrapper);
+      var wrapper = self._findWrapperInPath(e);
```

Comment:
I have used `e.target.parentNode` and it has been working so far. Though not very elegant (js code depending on dom structure) it is simple and seems to be working. Do you know if there is a problem with that?